### DISCUSSION TITLE
Vercel Deploy Error

### DISCUSSION THREAD

**Discussion by briankailey** [community/community#133616] [npm] [answered]
I have completed Next.js project, and then I am going to deploy that on **Vercel.**
But I got _node_modules install error_.
I fixed several times, but I didn't resolve that error yet.
How can I resolve that error.

**Comment by nexy39** [comment, marked as answer]
I see what you mean.
Plz activate and input in `Project/Settings/Build & Development Setting/Install Command `
_npm install --force_
Maybe, your error will be resolved
Good luck.

**Reply by briankailey** [reply]
Thanks for your kind advice.
I resolved that issue.
